src/test/java/nl/hsac/fitnesse/fixture/slim/FileFixtureTest.java

```diff
@@ -144,7 +144,10 @@ private String getExpectedUrlPath() {
     public void testDelete() {
         try {
             fixture.setDirectory(testResourcesDir);
+            fixture.copyTo(txtFilename, copyFilename);
             fixture.delete(copyFilename);
+        } catch (IOException ioe) {
```

Comment:
Please merge these two catch blocks into one. 
Best way probably is just to catch `Exception` instead of `SlimFixtureException`